docs/docs.css

```diff
@@ -104,7 +104,7 @@ html[dir="rtl"] .Markdown ul + pre {
   scroll-margin-top: 75px;
 }
 
-.Markdown h3 {
+.Markdown .mdx-header:not() h3 {
```

Comment:
This is hacky workaround around a color. Problem was that all of the H3s (###) when written in markdown would turn gray.  So I had to mark them with an "mdx-header" class to skip them.

If the headers don't absolutely have to be gray, and we could make them black, I don't need this workaround. Also, a code search for ### didn't reveal any h3s for me, so not sure if this part of the css is getting used

Reply:
I think the gray H3s were used on the old WHat's New changelog page. That gray styling be removed so this can be simpler